content/02.results.md

```diff
@@ -58,12 +44,40 @@ A review process identified samples that could not be explained parsimoniously,
 
 A further 82 samples were excluded as potential sample mix ups.
 
-Following all sample QC steps, 3,483 samples (74.2%) were retained from the original cohort for analysis.
+Following all sample QC steps, 3,483 wild-caught samples (74.2%) were retained from the original cohort for analysis.
+
+This represents an additional 1,823 mosquitoes relative to the phase 2 release.
+
+Due to a change in assessment of sample quality where technical replicates are available, the preferred replicate was changed for 172 mosquitoes between phase 2 and phase 3.
+
+9 samples included in phase 2 are not present in this release (sup XXX).
+
+The AG1000G-X submission set, made up of laboratory experimental crosses, was subject to a slightly different QC process.
+
+Firstly an analysis based on rates of Mendelian error identified true fathers of crosses (where multiple males were introduced to cages), and validated provided pedigrees.
+
+Of the 7XX samples provided we were able to validate 15 crosses to a high level of confidence, comprising 299 samples.
+
+4 of these crosses are novel relative to phase 2.
+
+These samples went through a modified sequence quality assurance process, where 1 sample was removed for insufficient coverage (methods).
+
+The final data release therefore comprises 3,XXX samples, XXX from laboratory crosses, and YYY wild collected samples.
 
 ### Coverage
 
+%% TO DO
+%% (PLOTS DONE, but numbers needed).
+
 Summary of site coverage post QC exclusions.
 
+- ie what frac of the genome is at 1X median
+- what frac at 10X. 
+- What frac of exome
+- what frac of X
+
+At this point we do not mention arabiensis.
+
 ### Species assignment and sex calling
 
 The Anopheles gambiae complex is a crypic group of sibling species, with no single locus offering unambiguous resolution of species.
```

Comment:
could we mention here that we think we have three species in these collections, gam, col, ara..? That then sets up the reason for why we need  methods to identify species.